tests/unit/models/test_model_interfaces.py

```diff
@@ -255,39 +255,37 @@ def _mock_data() -> tuple[tf.Tensor, tf.Tensor]:
 
 
 def _gpr(x: tf.Tensor, y: tf.Tensor) -> GPR:
-    return GPR((x, y), gpflow.kernels.Linear())
+    return GPR((x, y), gpflow.kernels.Matern32())
 
 
-def _sgpr(x: tf.Tensor, y: tf.Tensor) -> SGPR:
-    return SGPR((x, y), gpflow.kernels.Linear(), x[: len(x) // 2])
+def _sgpr(x: tf.Tensor, y: tf.Tensor, num_inducing: int = 2) -> SGPR:
+    return SGPR((x, y), gpflow.kernels.Matern32(), x[:num_inducing])
 
 
-def _svgp(inducing_variable: tf.Tensor) -> SVGP:
-    return SVGP(gpflow.kernels.Linear(), gpflow.likelihoods.Gaussian(), inducing_variable)
+def _svgp(x: tf.Tensor, y: tf.Tensor, num_inducing: int = 2) -> SVGP:
+    return SVGP(
+        gpflow.kernels.Matern32(), gpflow.likelihoods.Gaussian(), x[:num_inducing], num_data=len(x)
+    )
 
 
 def _vgp(x: tf.Tensor, y: tf.Tensor) -> VGP:
     likelihood = gpflow.likelihoods.Gaussian()
-    kernel = gpflow.kernels.Linear()
+    kernel = gpflow.kernels.Matern32()
     m = VGP((x, y), kernel, likelihood)
-    variational_variables = [m.q_mu.unconstrained_variable, m.q_sqrt.unconstrained_variable]
```

Comment:
how come this part can be removed?

Reply:
Before, when we were comparing VGP to GPR, we had to train the variational parameters before testing how similar it was. This is no longer necessary since we're comparing VGP to VGP.